source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,75 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+Comparison Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use comparison operations to compare two different values. You can
+perform a comparison operation on ``MqlValue`` objects using the methods
+described in this section.
+
+The following table lists the comparison methods
+available in the driver and corresponding expressions in the
+Query API. The methods link to API documentation and the Query API
+expressions link to Server manual documentation.
```

Comment:
S: "Server manual documentation" is redundant

```suggestion
expressions link to Server documentation.
```

Reply:
This is one of the copied sections.